packages/core/src/subscriber/chainSubscribers.ts

```diff
@@ -0,0 +1,181 @@
+            fromBlock: startFromBlock,
+            toBlock: 'latest',
+            topics: this.filterTopics,
+        });
+
+        const logs = rawLogs.sort((a, b) => {
+            if (a.blockNumber > b.blockNumber) {
+                return 1;
+            }
+            if (a.blockNumber < b.blockNumber) {
+                return -1;
+            }
+            // a must be equal to b
+            return 0;
+        });
+
+        // iterate
+        for (let x = 0; x < logs.length; x += 1) {
+            // verify if cusd or community and do things
+            await this._filterAndProcessEvent(provider, logs[x]);
+        }
+        utils.Logger.info('Past events recovered successfully!');
+    }
+
+    _setupListener(provider: ethers.providers.JsonRpcProvider) {
+        utils.Logger.info('Starting subscribers...');
+        const filter = {
+            topics: this.filterTopics,
+        };
+
+        provider.on(filter, async (log: ethers.providers.Log) => {
+            await this._filterAndProcessEvent(provider, log);
+            services.app.ImMetadataService.setLastBlock(log.blockNumber);
+        });
+    }
+
+    async _filterAndProcessEvent(
+        provider: ethers.providers.JsonRpcProvider,
+        log: ethers.providers.Log
+    ) {
+        let parsedLog: ethers.utils.LogDescription | undefined;
+        if (log.address === config.communityAdminAddress) {
+            await this._processCommunityAdminEvents(log);
+        }
+        return parsedLog;
+    }
+
+    async _processCommunityAdminEvents(
+        log: ethers.providers.Log
+    ): Promise<ethers.utils.LogDescription | undefined> {
+        const parsedLog = this.ifaceCommunityAdmin.parseLog(log);
+        let result: ethers.utils.LogDescription | undefined = undefined;
+
+        if (parsedLog.name === 'CommunityRemoved') {
+            const communityAddress = parsedLog.args[0];
+            const community = await database.models.community.findOne({
+                attributes: ['publicId'],
+                where: { contractAddress: communityAddress },
+            });
+
+            if (!community || !community.publicId) {
+                utils.Logger.error(
+                    `Community with address ${communityAddress} wasn't found at "CommunityRemoved"`
+                );
+            } else {
+                await database.models.community.update(
+                    {
+                        status: 'removed',
+                        deletedAt: new Date(),
+                    },
+                    {
+                        where: { contractAddress: communityAddress },
+                    }
+                );
+                await database.models.beneficiary.update(
+                    {
+                        active: false,
+                    },
+                    {
+                        where: {
+                            communityId: community.publicId,
+                        },
+                    }
+                );
```

Comment:
You can remove this. We don't consume from this table anymore and it will be dropped.